src/hssm/hssm.py

```diff
@@ -1619,24 +1671,103 @@ def _postprocess_initvals_deterministic(
         # Consider case where link functions are set to 'log_logit'
         # or 'None'
         if self.link_settings not in ["log_logit", "None", None]:
```

Comment:
We do not allow `link_settings` to be a string `"None"` per type definition of argument